apps/speed.c

```diff
@@ -768,28 +768,29 @@ static int EVP_Update_loop_ccm(void *args)
     EVP_CIPHER_CTX *ctx = tempargs->ctx;
     int outl, count;
     unsigned char tag[12];
+    int dummy = 1;
 
     if (decrypt) {
         for (count = 0; COND(c[D_EVP][testnum]); count++) {
             (void)EVP_CIPHER_CTX_ctrl(ctx, EVP_CTRL_AEAD_SET_TAG, sizeof(tag),
                                       tag);
             /* reset iv */
-            (void)EVP_DecryptInit_ex(ctx, NULL, NULL, NULL, iv);
+            dummy *= EVP_DecryptInit_ex(ctx, NULL, NULL, NULL, iv);
```

Comment:
Does `+=` or `|=`work instead?  Multiplication is expensive on some platforms.

Reply:
Resolving outdated conversation